presto-main/src/main/java/com/facebook/presto/operator/LookupJoinPageBuilder.java

```diff
@@ -14,18 +14,146 @@
 package com.facebook.presto.operator;
 
 import com.facebook.presto.spi.Page;
+import com.facebook.presto.spi.PageBuilder;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.type.Type;
+import it.unimi.dsi.fastutil.ints.IntArrayList;
 
-public interface LookupJoinPageBuilder
+import java.util.List;
+
+import static com.facebook.presto.spi.block.PageBuilderStatus.DEFAULT_MAX_PAGE_SIZE_IN_BYTES;
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Verify.verify;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * This page builder creates pages with dictionary blocks:
+ * normal dictionary blocks for the probe side and the original blocks for the build side.
+ *
+ * TODO use dictionary blocks (probably extended kind) to avoid data copying for build side
+ */
+public class LookupJoinPageBuilder
```

Comment:
Consider doing `LookupJoinPageBuilder` interface removal in the previous commit, where you delete the other implementation and doing rename separately. Otherwise GH cannot show meaningful diff